specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,10340 @@
+    "/datasources/{dataSourceName}/scans/{scanName}/triggers/default:enable": {
+      "post": {
+        "tags": [
+          "Triggers"
+        ],
+        "description": "Enables a trigger.",
+        "operationId": "Triggers_Enable",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "dataSourceName",
+            "required": true,
+            "type": "string",
+            "description": "data source name"
+          },
+          {
+            "in": "path",
+            "name": "scanName",
+            "required": true,
+            "type": "string",
+            "description": "scan name"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/Trigger"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Triggers_Enable": {
+            "$ref": "./examples/Triggers_Enable.json"
+          }
+        }
+      }
+    },
+    "/datasources/{dataSourceName}/scans/{scanName}/triggers/default:disable": {
+      "post": {
+        "tags": [
+          "Triggers"
+        ],
+        "description": "Disables a trigger.",
+        "operationId": "Triggers_Disable",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "dataSourceName",
+            "required": true,
+            "type": "string",
+            "description": "data source name"
+          },
+          {
+            "in": "path",
+            "name": "scanName",
+            "required": true,
+            "type": "string",
+            "description": "scan name"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/Trigger"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Triggers_Disable": {
+            "$ref": "./examples/Triggers_Disable.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AzureKeyVaultProperties": {
+      "type": "object",
+      "properties": {
+        "baseUrl": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        }
```

Comment:
I see that you've added descriptions to many properties, but there are still many, such as these two, with no description. This makes the API difficult to use for developers. Please make another pass on this.

Reply:
Just added description of this part and others